ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java

```diff
@@ -943,6 +947,12 @@ private int countChecked(Object[] elements) {
 	}
 
 	public void performApply(ILaunchConfigurationWorkingCopy config) {
+		if (!fControlCreated) {
+			// This method is also being called before the proper UI elements
+			// are created, which might cause a NPE
+			return;
+		}
```

Comment:
FYI this one is to prevent the NPE mentioned in [my comment](https://github.com/eclipse-pde/eclipse.pde/pull/1233#issuecomment-2061789215)